frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -14,22 +14,53 @@
  * limitations under the License.
  */
 
-import { Component, h } from 'preact';
-import { translate } from 'react-i18next';
-import { FiatConversion } from '../rates/rates';
+import { Component, h, RenderableProps } from 'preact';
+import { translate, TranslateProps } from '../../decorators/translate';
+import { Fee } from '../../routes/account/send/send';
```

Comment:
@benma I am not sure about importing the interface from `../../routes/account/send/send'` I renamed it  in send, is that ok?

Reply:
importing interfaces is fine, but commented on the naming in the other comment.